development/build2/build2.SlackBuild

```diff
@@ -83,19 +79,21 @@ find -L . \
  \( -perm 666 -o -perm 664 -o -perm 640 -o -perm 600 -o -perm 444 \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
+# Fix chroot folder (to avoid having /tmp/SBo/package-* in the resulting files)
```

Comment:
It looks like something is not quite right with your patch. At the end it is trying to run `b` which is part of this package. It fails there.

```
install bdep/legal{AUTHORS} -> ../package-build2/usr/share/doc/bdep/
install bdep/legal{LEGAL} -> ../package-build2/usr/share/doc/bdep/
install bdep/manifest{manifest} -> ../package-build2/usr/share/doc/bdep/
+ command -v b
```

You might want to check how arch handle it:

https://gitlab.archlinux.org/archlinux/packaging/packages/build2/-/blob/main/PKGBUILD

Reply:
Did you test this locally? It fails in the same place for me just checking out this branch and running the slackbuild.